string action 
int64 task_id
string component
string status
string coordinate